gradle.properties

```diff
@@ -13,6 +13,7 @@ dockerExecutable=/usr/bin/docker
 #3rd party Versions
 apacheCommonsCsvVersion=1.5
 apacheCuratorVersion=4.0.1
+apacheZookeeperVersion=3.5.5
```

Comment:
Is it safe to make this change? If we are not running the 3.5.5 server, then I'm not convinced it is safe.

Reply:
Now tested with Zookeeper 3.5.5 server as well.